packages/webui/src/client/styles/main.scss

```
@@ -85,6 +85,7 @@ input {
 @import 'shelf/timeOfDayPanel';
 
 @import '../lib/forms/SchemaFormTable/ObjectTable.scss';
+@import '../lib/forms/SchemaFormOneOfButtons/OneOfButtons.scss';
```

Stylelint (17.6.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)